Libplanet/Net/Swarm.cs

```diff
@@ -545,6 +541,40 @@ IEnumerator IEnumerable.GetEnumerator()
                 cancellationToken);
         }
 
+        /// <summary>
+        /// Synchronizes the blocks with other registerd <see cref="Peer"/>s.
+        /// </summary>
+        /// <param name="blockChain">
+        /// A <see cref="BlockChain{T}"/> instance to synchronize.
+        /// </param>
+        /// <param name="progress">
+        /// An instance that receives progress updates for block downloads.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.
+        /// </param>
+        /// <typeparam name="T">An <see cref="IAction"/> type. It should match
+        /// to <see cref="BlockChain{T}"/>'s type parameter.</typeparam>
+        /// <returns>
+        /// No object or value is returned by this method when it completes.
+        /// </returns>
+        public async Task PreloadAsync<T>(
+            BlockChain<T> blockChain,
```

Comment:
This parameter seems common in methods in `Swarm`.  Is it inappropriate to have a field for this instead?

Reply:
It's also appropriate, but I want to keep `Swarm`  as `T` agnostic, so I only get arguments for methods that use block chains.

From another point of view, it would be better to refactor by dividing into the appropriate classes because `Swarm` is currently too large. I'll create an issue about this.

Reply:
Yes, it's anyway a refactoring.  We'd better have a separated patch for that.